level-1/삼총사.py

```diff
@@ -0,0 +1,12 @@
+def solution(number):
+    answer = 0
+    total = 0
+    
+    for i in range(len(number)):
+        for j in range(len(number)):
+            for k in range(len(number)):
+                if i != j and j != k and k != i and number[i] + number[j] + number[k] == 0:
```

Comment:
LGTM! What I meant for elegant was  using `i < j  and j < k`, so that we don't have to divide the final result by 6, but whatever. Keep up the good work!